topi/python/topi/broadcast.py

```diff
@@ -106,6 +106,7 @@ def _bcast_to_arg_eval(data, bcast_info, *indices):
                 indices_tuple.append(0)
         return data[tuple(indices_tuple)]
     original_shape = data.shape
+    shape = [i if isinstance(i, int) else get_const_int(i) for i in shape]
```

Comment:
i think this is the case where we need to update get_const_int instead to handle int